docs/terms/store.md

```diff
@@ -3,19 +3,107 @@ title: Store
+
+## Features
+
+### Metadata storage and retrieval
+
+Stores permit the configuration of where certain types of data and metadata will be stored and retrieved from.
 
-## Finding your Stores
+### Centralized access and convenient API
 
-You can find a list of all of your Stores in your `great_expectations.yml` file.  Alternatively, from the root folder of your Data Context you can retrieve a list of Stores with a CLI command:
+Stores provide an API for Great Expectations to perform the retrieval and storage of information.  This API (e.g. the Stores themselves) is always available from your Data Context.
+
+## API Basics
+
+### How to access
+
+You can find a list of almost all of your Stores in your `great_expectations.yml` file under the `stores` key.  Alternatively, from the root folder of your Data Context you can retrieve a list of Stores with a CLI command:
 
 ```markdown title="Console command"
 great_expectations store list
 ```
 
+Data Docs Stores are the one exception to the above.  They are instead configured under the `data_docs_sites` key in the `great_expectations.yml` file.  Each entry under `data_docs_sites` will have a `store_backend` key.  The information under `store_backend` will correspond to the Data Doc Store used by that site. Data Doc Stores are not listed by the `great_expectations store list` command.
+
+### Configuration
+
+Configuration of your Stores is located in your `great_expectations.yml` file under the `stores` key.  Three stores are required: an Expectation Store, a Validation Result Store, and an Evaluation Parameter Store must always have a valid entry in the `great_expectations.yml` file.  Additional Stores can be configured for uses such as Data Docs and Checkpoints (and a Checkpoint Store configuration entry will automatically be added if one does not exist when you attempt to save a Checkpoint for the first time.)
+
+Note that unlike other Stores, your Data Docs Stores are configured under each individual site under the `data_docs_sites` key in the `great_expectations.yml` file.
+
+## More details
+
+For more information about a specific type of Store, please see the Store type's technical term definition page:
+
+- [Expectation Store technical term page](./expectation_store.md)
+- [Checkpoint Store technical term page](./checkpoint_store.md)
+- [Validation Result Store technical term page](./validation_result_store.md)
+- [Evaluation Parameter Store technical term page](./evaluation_parameter_store.md)
+- [Data Docs Store technical term page](./data_docs_store.md)
+- [Metric Store technical term page](./metric_store.md)
+
+
+
+
+
+
+
+
+
+
+
 
 
 NOTES: TEMPORARY
```

Comment:
either remove this section or rename the header 😄

Reply:
Haha.  Thanks for the catch.  Clearly I meant to go back to this page after creating the other ones and then... forgot. >.>